semtypes/src/main/java/io/ballerina/semtype/subtypedata/FloatSubtype.java

```diff
@@ -36,18 +35,16 @@
  */
 public class FloatSubtype extends EnumerableSubtype implements ProperSubtypeData {
 
```

Comment:
We are not gaining anything by moving this to base class right. All I see is not our implementation is different from nBallerina. shall we not do this please

Reply:
Sorry I'm bit late to respond to your comment

Reply:
np, shall we use getter methods to access the fields in that case since otherwise the parent classes variable will be accessed?

Reply:
Do we need fields in EnumerableSubtype?

Reply:
No can just keep in subtypes and use abstract accessor methods in EnumerableSubtype

Reply:
+1